modules/retail/pages/retail-deploy-terminals.adoc

```diff
@@ -0,0 +1,116 @@
+After you have created your new hardware type group, you can apply the [systemitem]``saltboot-formula`` to the group and configure it for your environment.
+
+Hardware types allow you to group devices according to manufacturer and device name, so that all devices of a particular type can be managed as one.
+
+You will need to create appropriate hardware types in your system.
+
+Empty profiles can be assigned to a hardware type group either before or after registration.
+If an empty profile is not assigned to a hardware type group before registration, it will be assigned to group that best matches the product information available to it.
+
+For this procedure, you will require the system manufacturer name and product name for your terminal.
+
+.Procedure: Creating a Hardware Type group
+
+. Determine the hardware type group name.
+Prefix the name with [systemitem]``HWTYPE:``, followed by the system manufacturer name and product name, separated by a hyphen.
+For example:
++
+----
+HWTYPE:POSVendor-Terminal1
+----
++
+[IMPORTANT]
+====
+Only use colons (``:``), hyphens (``-``) or underscores (``_``) in hardware type group names.
+Spaces and other non-alphanumeric characters will be removed when the name is processed.
+====
++
+. In the {susemgr} {webui}, navigate to menu:Systems[System Groups], and click the btn:[Create Group] button.
+. In the [guimenu]``Create System Group`` dialog, create a new system group, using the hardware type group name you created earlier.
+
+
+== Assign and Configure the Saltboot formula for each hardware type group
+
+Each hardware type group must have the Saltboot formula applied.
+
+.Procedure: Assigning the Saltboot formula
+
+. Open the details page for your new hardware type group, and navigate to the [guimenu]``Formulas`` tab.
+. Select the [systemitem]``saltboot-formula`` and click btn:[Save].
+. Navigate to the new menu:Formulas[Saltboot] tab.
+. Configure the Saltboot formula according to the instructions in xref:retail:retail-formulas.adoc[].
+
+
+== Synchronize images to the branch server
+
+.Procedure: Synchronizing images to the branch server
+
+. On the {susemgr} server, run this command:
++
+----
+salt <branch_server_salt_id> state.apply image-sync
+----
+
+
+=== Using a 32-bit default boot image
+
+If you have 32-bit machines included in your branch, then you must use a 32-bit default boot image.
+
+You can check the available boot images and their architecture from the command line:
+
+----
+# salt "branch" pillar.get boot_images
+
+POS_Image_JeOS6-6.0.0:
+      ----------
+      arch:
+          x86_64
+      ...
+legacy-6.0.0:
+      ----------
+      arch:
+          i686
+----
+
+In this example, the [systemitem]``legacy-6.0.0`` boot image is 32-bit.
+
+You can set the default boot image in the Image Synchronization formula, by adding the chosen boot image name to the [guimenu]``Synchronize only the listed images`` field.
```

Comment:
Boot image name has to be added to `Default boot image` field. (This feature is new in RC1.) Formula itself is related to branch server configuration only, maybe we should notice that. Please see http://imagebin.suse.de/3235/img.